doc/manual/install-judgehost.rst

```diff
@@ -179,16 +179,9 @@ Optionally the timings can be made more stable by not letting the OS schedule
 any other tasks on the same CPU core the judgedaemon is using:
 ``GRUB_CMDLINE_LINUX_DEFAULT="quiet cgroup_enable=memory swapaccount=1 isolcpus=2"``
 
-On modern distros (e.g. Debian bullseye and Ubuntu Jammy Jellyfish) which have
-cgroup v2 enabled by default, you need to add ``systemd.unified_cgroup_hierarchy=0``
-as well. Then run ``update-grub`` and reboot.
-After rebooting check that ``/proc/cmdline`` actually contains the
-added kernel options. On VM hosting providers such as Google Cloud or
-DigitalOcean, ``GRUB_CMDLINE_LINUX_DEFAULT`` may be overwritten
-by other files in ``/etc/default/grub.d/``.
-
 You have now configured the system to use cgroups. To create
-the actual cgroups that DOMjudge will use, run::
+the actual cgroups that DOMjudge will use, on systems with cgroups v1, you need
+to run::
```

Comment:
```suggestion
to run:
```

Reply:
`::` is used in RST to indicate literal blocks: https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html#literal-blocks

Reply:
> When text immediately precedes the "::", one colon will be removed from the output, leaving only one colon visible (i.e., "::" will be replaced by ":"; this is the "fully minimized" form).

Not really intuitive, but sure :joy: Thanks, learned something new today :smile: